language-examples/csharp/grpc/GrpcBasicExample.cs

```diff
@@ -129,12 +134,18 @@ private static String Index(long customerId, long corpusId, String indexingEndpo
                 });
 
                 var result = indexingClient.Index(request);
-                Console.WriteLine(result.ToString());
+                if (result.Status.Code == Com.Vectara.StatusCode.Ok) {
+                    Console.WriteLine("Document indexed successfully.");
+                } else if (result.Status.Code == Com.Vectara.StatusCode.AlreadyExists) {
+                    Console.WriteLine("Document was previously indexed.");
```

Comment:
nit: perhaps add that upserts are not currently supported (or just append `result.Status.StatusDetail` to the log message).

Reply:
Appended.